### PR TITLE
package novice replaced with good first issue

### PR DESCRIPTION
**Pull request by mridullpandey**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -84,7 +84,7 @@ The main one is the SunPy repository with where all the known `issues`_ with Sun
 Each issue should have a series of labels that provide information about the nature of the issue.
 If you find an issue you'd like to work on, please make sure to add a comment to let people know that you are working on it! This will make it less likely that effort is duplicated.
 
-If you are unsure where to start we suggest the `Package Novice label`_.
+If you are unsure where to start we suggest the `Good First Issue`_.
```
Comment (nabobalis) [inline CONTRIBUTING.rst:87]:
```suggestion
If you are unsure where to start we suggest the `Good First Issue`_ label.
```

You also need to change the URL and name of this at line 113.

**Comment by nabobalis** [comment]
Can you update your branch? Hopefully the checks will pass when that happens.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (mridullpandey) [reply CONTRIBUTING.rst:87]: https://github.com/sunpy/sunpy/issues?utf8=✓&q=is:issue+is:open+label:"Good+First+Issue" will this work

**Comment by nabobalis** [comment]
**FOR MAINTAINERS** Issues need to be updated before this can be merged.

**Comment by nabobalis** [comment]
Thanks @mridullpandey

**Pull request merged**